GAE/src/resources/caddisfly/caddisfly-tests.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
AFAIK, this is a shared resource between the app and the server. Shouldn't we include some time of `"version"` property so we know we're using the same file? I'm not completely sure what are the implications of having out-of-sync files (e.g. the app behind the server)